changes/serializers.py

```diff
@@ -50,7 +50,7 @@ class ReceiveWhatsAppEventSerializer(serializers.Serializer):
     class StatusSerializer(serializers.Serializer):
 
         class ErrorSerializer(serializers.Serializer):
-            code = serializers.CharField(required=False)
+            code = serializers.IntegerField(required=False)
```

Comment:
We'll probably need a migration for this?

Reply:
Not for a serializer, do we?

Reply:
Oh sorry, thought this was a model for some strange reason, 👍